tests/AutoReview/FixerFactoryTest.php

```diff
@@ -260,12 +260,13 @@ public function testFixersPriorityPairsHaveIntegrationTest(FixerInterface $first
 
     public function provideFixersPriorityPairsHaveIntegrationTestCases()
     {
+        $self = $this;
```

Comment:
Not sure if there's a convention on how to do this on PHP 5.3, but I myself have used (and seen) others use

```php
$that = $this;
```

Maybe using `$self` is a bit confusing?

Reply:
I saw both conventions, no hard feeling about it

Reply:
`var that = this` is common in Javascript. Also, `$self` could be confused with `self`. I would go for `$that`.

Reply:
> `var that = this` is common in Javascript

`that` is disrespected by Crockford, he asked you to use self ;P

we gonna drop it in very next branch...

Reply:
:+1: for `$self` , 
I get confused when `this` is referred by `that`

Reply:
guys, let go anyway here, I'm fine with both, if any of you wants - let you just change it yourself ;)
Please, let us focus on real idea behind this PR instead ;)